cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceProgressUtils.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.assembly;
+
+/**
+ * Utility class for handling progress fields of KafkaRebalance custom resource
+ */
+public class KafkaRebalanceProgressUtils {
+
+    private static final int SECONDS_IN_A_MINUTE = 60;
```

Comment:
I mention this below, but this it smells wrong. Unless you need the elapsed time / data rate in terms of seconds just use the elapsed time in minuets throughout. Else we could return a value in terms of seconds (to be parsed by user tooling) **AND** a human readable ("HH:MM:SS") field?

Reply:
Since moving the conversion to minutes on the same line as the original rate calculation we probably don't need this field anymore.